addons/l10n_hr_edi/models/l10n_hr_edi_addendum.py

```diff
@@ -0,0 +1,66 @@
+from odoo import models, fields
+
+
+class L10nHrEdiAddendum(models.Model):
+    _name = 'l10n_hr.edi_addendum'
+    _description = 'EDI and fiscalization information for Croatian electronic invoicing'
+
+    move_id = fields.Many2one('account.move', required=True, ondelete='cascade', index=True)
+
+    # EDI and fiscalization-specific fields
+    l10n_hr_invoice_sending_time = fields.Datetime(string="Time of invoicing")
+    l10n_hr_business_document_status = fields.Selection(
+        selection=[
+            ('0', 'APPROVED'),
+            ('1', 'REJECTED'),
+            ('2', 'PAYMENT_FULFILLED'),
+            ('3', 'PAYMENT_PARTIALLY_FULLFILLED'),
+            ('4', 'RECEIVING_CONFIRMED'),
+            ('99', 'RECEIVED'),
+            ('None', 'None')
+        ],
+        string='Business document status',
+    )
+    l10n_hr_business_status_reason = fields.Char(
+        string='Document rejection reason',
+        default='None',
+    )
+    l10n_hr_fiscalization_number = fields.Char(
+        string='Invoice fiscalization number',
+    )
+    l10n_hr_fiscalization_status = fields.Selection(
+        selection=[
+            ('0', "Successful"),
+            ('1', "Unsuccessful"),
+            ('2', "Pending"),
+        ],
+        string='Fiscalization status',
+    )
+    l10n_hr_fiscalization_error = fields.Char(
+        string='Error reported for fiscalization',
+        default='None',
+    )
+    l10n_hr_fiscalization_request = fields.Char(
+        string='Fiscalization request ID',
+    )
+    l10n_hr_fiscalization_channel_type = fields.Selection(
+        selection=[
+            ('0', "Delivered via EDI"),
+            ('1', "Not delivered via EDI"),
+        ],
+        string='Delivery channel type',
+        help="If delivery via EDI fails, the invoice is reported to tax authorities but has to be delivered to the client via other means (ex. e-mail)."
+    )
+    # MojEracun integration fields
+    l10n_hr_mer_document_id = fields.Char(string='MojEracun document ElectronicId', copy=False)
```

Comment:
```suggestion
    l10n_hr_mer_document_ident = fields.Char(string='MojEracun document ElectronicId', copy=False)
```

Let's rename this field because as a convention in Odoo, `_id` fields are `Many2one`s.